include/OCPNPlatform.h

```diff
@@ -128,7 +128,25 @@ class OCPNPlatform
     wxString &GetSharedDataDir();
     wxString &GetPrivateDataDir();
     wxString GetWritableDocumentsDir();
+
+    /** The original in-tree plugin directory, sometimes not user-writable.*/
     wxString &GetPluginDir();
+
+    /**
+     * Base directory for user writable windows plugins, reflects
+     * winPluginDir option, defaults to %LOCALAPPDATA%/opencpn.
+     **/
+    wxString &GetWinPluginBaseDir();
```

Comment:
Should this be `GetUserPluginBaseDir()`? On Linux with OpenCPN in /usr/local does it return something under ${HOME}/.local?

Reply:
Not really, this is the windows stuff. It might be possible to create  common routine later, but already standard linux is different enough to make it complicated. And there is still MacOS and flatpak... I simply don't know at this stage.

Reply:
Expanding on above: the windows single locations is thought as configurable, Dave insisted that this is important.  OTOH are the Linux multiple locations basically hardcoded, but could be changed using the environment.

Reply:
Fair enough. You understand this much better than I do.

I was thinking that most platforms have "global" (=where opencpn is installed) and "user" (=${HOME}/.local / %LOCALAPPDATA% / ...) directory, of course what they are and how they are guessed differs. And for portable mode they reduce to the same directory. Thus even if the User location is hard coded on Linux at the moment using the same API would have made sense.

Feel free to ignore this.

Reply:
It's not about ignoring, you have a point. But I feel that before making these kind of generalizations we need to understand the flatpak/MacOS usecases better.

Reply:
see below on dirs used to load from vs dir used to install in.